src/zope/publisher/tests/test_browserrequest.py

```diff
@@ -220,6 +227,38 @@ def testFileUploadPost(self):
         # Test that we can actually read the file data
         self.assertEqual(request.form['upload'].read(), b'Some data')
 
+    def testFileUploadPost(self):
+        """Produce a Fieldstorage with a file handle that exposes
+        its filename."""
```

Comment:
The default `unittest` runner still prints the docstrings, even in Python 3 (at least up to 3.4).